### PR TITLE
Feature/react webapp

### PR DESCRIPTION
**Pull request by marcofranssen**

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -1,104 +1,36 @@
...
-    "replace": "^1.0.0",
-    "rxjs": "^6.2.2",
-    "ts-node": "^4.1.0",
-    "tslint": "^5.11.0",
-    "typescript": "^2.9.2",
-    "wait-on": "^2.1.0",
-    "webdriver-manager": "^12.0.6",
-    "zone.js": "^0.8.26"
-  }
+    "electron": "^3.0.8",
```
[the remark this answers is not in the document]
Comment (marcofranssen) [reply web/package.json]: See readme.md First run `npm run build`

File: web/src/pages/Donate/Donate.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { pageStyles } from '../styles';
+
+const styles = theme => ({
+  ...pageStyles(theme)
+});
+
+const donationOptions = [
+  {
+    currency: 'BTC',
+    address: '1LpbqH52qSAZaTsYqsY9Stoxx1Ta7n5UD5'
```
[the remark this answers is not in the document]
Comment (marcofranssen) [reply web/src/pages/Donate/Donate.js]: Wouldn't it be great to have more ways to get donations? Will update the BTC one soon

**Comment by gloriousCode** [comment]
Thanks for making those changes!

A few more things have com up:
npm run electron is having issues with the pathing:
image

npm run electron-dev does not render:
image

**Comment by marcofranssen** [comment]
See the Readme file in the web folder. For electron dev you should do npm start before.

For the other on npm run build as documented in Readme.

**Comment by gloriousCode** [comment]
Alright with the npm start for dev. The first screen was with npm run build -> npm run electron still has the pathing issues

**Comment by marcofranssen** [comment]
Let me check that tonight after work. Will have some time then.

**Comment by marcofranssen** [comment]
I have asked for help on that pathing issue in following places.

https://github.com/facebook/create-react-app/pull/1718
https://medium.freecodecamp.org/building-an-electron-application-with-create-react-app-97945861647c

On the other hand we can continue development also on the browser, so do we need to wait for the electron fix?

**Pull request merged**